package.json

```diff
@@ -28,7 +28,7 @@
         "test": "bash bin/test",
         "fmt-check": "bash bin/fmt-check",
         "fmt": "bash bin/fmt",
-        "postinstall": "bash bin/postinstall"
+        "postinstall": "bin/postinstall"
```

Comment:
Not sure if this would actually work in windows. In worst case would need to use `"postinstall": "node bin/postinstall"`